src/pages/User/components/PullRequests/index.js

```diff
@@ -87,9 +88,14 @@ class PullRequests extends Component {
           )
       );
 
-      const [data, userDetail] = await Promise.all(allResponses);
+      const [data, userDetail, lFMembershipStatus] = await Promise.all(allResponses);
       const count = this.counterOtherRepos(data, userDetail);
 
+      // if user is a member of Leapfrog then there will no any response
+      if (lFMembershipStatus !== undefined) {
+        alert('Aw snap! You are not a member of Leapfrog Technology!');
```

Comment:
Showing a classic `alert` is not the a neat solution; neither it is good from UX perspective. You can create a component to show this in the UI as a message along with the link below.